src/driver/mongodb.rs

```diff
@@ -0,0 +1,261 @@
+extern crate bson;
+extern crate mongodb;
+
+use std::error::Error as ErrorTrait;
+use std::ops::Deref;
+use linear_map::LinearMap;
+use self::bson::{Bson, Document};
+use self::mongodb::{Client, ThreadedClient, Error as MongoDBError};
+use self::mongodb::connstring;
+use self::mongodb::db::{Database, ThreadedDatabase};
+use schema::{Definition, Schema};
+use driver::Driver;
+use error::{Error, ErrorCode};
+use patch::Patch;
+use query::{Query, Selection};
+use value::Value;
+
+struct MongoDBDriver {
```

Comment:
True, but this sets a precedent. PostgreSQL -> `postgres`, CouchDB -> `couch`, MariaDB -> `maria`. I don't think this is a bad thing, but something to think about nontheless.